parts/linux/cloud-init/artifacts/nvidia-persistenced.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=NVIDIA Persistence Daemon
+Wants=syslog.target
+
+[Service]
+Type=forking
+ExecStart=/usr/bin/nvidia-persistenced --verbose
+ExecStopPost=/bin/rm -rf /var/run/nvidia-persistenced
+PIDFile=/var/run/nvidia-persistenced/nvidia-persistenced.pid
```

Comment:
self-nit: is the pid file usable if it's deleted by ExecStopPost? which happens first?

Reply:
I got rid of this, it's not in upstream template any more